### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by tropez67**
Adding APC AP8841 device type

### CODE REVIEW

File: device-types/APC/AP8841.yaml [hunk 1]
```diff
@@ -0,0 +1,184 @@
+---
+manufacturer: APC
+model: AP8841
+slug: apc-ap8841
+part_number: AP8841
+u_height: 0
+is_full_depth: false
+comments: Rack PDU 2G, metered, 0U, 30A, 200/208V, (36) C13 & (6) C19. (2) 20A 2-pole hydraulic-magnetic circuit breakers
```
Comment (danner26) [inline device-types/APC/AP8841.yaml:8]: Please add the `airflow` data point if you have the information

Comment (danner26) [inline device-types/APC/AP8841.yaml:8]: Please add the `weight` and `weight_unit` data point if you have the information

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/APC/AP8841.yaml (hunk 1 above)
Comment (tropez67) [reply device-types/APC/AP8841.yaml:11]: This particular unit has no fan for any airflow.

Comment (tropez67) [reply device-types/APC/AP8841.yaml:11]: I've added the weight and weight_unit.

Comment (danner26) [reply device-types/APC/AP8841.yaml:11]: Then the data point should be `airflow: passive`

**Comment by danner26** [comment]
After adding the airflow attribute, please make sure this commit passes all tests
The easiest way to do so is to utilize `pre-commit` as explained in the readme. If you need assistance please let me know!

**Comment by danner26** [comment]
Hello, we have released a new set of tests and features to assist with properly creating definition files for the DeviceType-Library. These tests will ensure that all files conform to a standardized format and contain valid data. Over the coming weeks we will be adding more tests to handle other currently undefined cases. If you notice anything that you believe is a bug, please @ me in a comment and details what you think is incorrect.

Along with additional test cases we have added a feature that will annotate why tests fail. To access these annotations after a test has completed, please navigate to the _Files changed_ tab within your PR. At the very bottom of the page you will see a list of errors along with the filename in question and the error which was found.

With all this being said, if your PR is validated then great! Please @ me me so I can review it further! If your PR is not valid, please review the issues and correct them; in the meantime I will mark your PR as a draft. After correcting the issues and recieving a valid status, please mark the PR as ready to be reviewed. Thank you!

**Pull request closed without merging**